quasar-utils/javac-plugin/src/main/java/co/paralleluniverse/fibers/Suspendable.java

```diff
@@ -0,0 +1,9 @@
+package co.paralleluniverse.fibers;
+
+import java.lang.annotation.*;
+
+@Documented
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD, ElementType.TYPE})
+public @interface Suspendable {
+}
```

Comment:
This is another Quasar class. Are you also aware that we have recently modified Quasar so that Corda can declare its own `@Suspendable` annotation? CorDapp developers are not going to be using this annotation any more.

Reply:
I wasn't aware of that (and to be honest I don't even understand what's the advantage of not using `@Suspendable` since corda and cordapp code will still depends on quasar either way). I will address it

Reply:
The point is that _the API we expose to CorDapps_ will no longer contain either Quasar or its army of transitive dependencies. Obviously Corda will continue to use Quasar internally.

More specifically, `quasar-core-0.8.x` has been modified so that you can add the class names of _additional_ `@Suspendable` annotations. Quasar's built-in `@Suspendable` annotation is still supported regardless. `qusar-utils-6.0.0-SNAPSHOT` now has a new
```groovy
Property<String> suspendableAnnotation
```
on its `quasar` extension to configure Quasar with Corda's own `@Suspendable` annotation.

Corda 5.x will use `quasar-core-0.8.x`. Earlier versions of Corda use `quasar-core-0.7.x`, which still only supports Quasar's built-in `@Suspendable` annotation.

Reply:
imho, it would have been simpler to just extract `@Suspendable` and `SuspendExecution` to a separate `quasar-annotations` module, with `quasar-core` that depends on `quasar-annotations` but thanks for the clarification anyway

Reply:
I guess you will now have to agree with the node, when you install a new cordapp, about which annotation is meant to mark a suspendable method

Reply:
> I guess you will now have to agree with the node, when you install a new cordapp, about which annotation is meant to mark a suspendable method in a CorDapp

Like I said, Corda will provide an _extra_ `@Suspendable` annotation, and Corda 5 CorDapps will be expected to use it. Quasar will continue to support its built-in `@Suspendable` annotation too, and so there shouldn't be any scope for disagreement between CorDapps and the node.

To be honest, I initially thought that you had asked @lankydan to review this PR because you were already aware of the changes currently being made to Quasar.

Reply:
Everything @chrisr3 said is correct, we now have our own `@Suspendable` annotation, I _think_ its in `core` but either way its in our code base.

Reply:
> Everything @chrisr3 said is correct, we now have our own `@Suspendable` annotation, I _think_ its in `core` but either way its in our code base.

It's actually in `:base`, but I'll let you off :wink:.